.pre-commit-config.yaml

```diff
@@ -11,6 +11,14 @@ repos:
       - id: trailing-whitespace
         exclude: ^frontend2/src/utils/types
 
+  # This is only used for the old frontend. The prettier check for frontend2 exists in ci.yml
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
+        types_or: [css, html, javascript, json, jsx, markdown]
+        exclude: ^(frontend/public/assets|frontend2)
```

Comment:
Excluding frontend2 -- why do we need this then?

Also- pre-commit is useful for catching things locally before they get rejected by CI, so may recommend keeping if it's DRY.

Reply:
this is in prep for merging into main, so that the old frontend still has linting checks

I excluded the frontend2 folder because frontend2 has a prettier tailwindcss extension and I can't get it to work with precommit (which is why frontend2 prettier checks are in ci.yml instead of precommit)